src/platform_impl/windows/window.rs

```diff
@@ -74,19 +105,20 @@ impl Window {
         unsafe {
             let drag_and_drop = pl_attr.drag_and_drop;
             init(w_attr, pl_attr, event_loop, |win| {
-                let file_drop_handler = if drag_and_drop {
-                    use winapi::shared::winerror::{OLE_E_WRONGCOMPOBJ, RPC_E_CHANGED_MODE, S_OK};
-
-                    let ole_init_result = ole2::OleInitialize(ptr::null_mut());
+                let drop_target = if drag_and_drop {
+                    let ole_init_result = OleInitialize(ptr::null_mut());
                     // It is ok if the initialize result is `S_FALSE` because it might happen that
                     // multiple windows are created on the same thread.
-                    if ole_init_result == OLE_E_WRONGCOMPOBJ {
-                        panic!("OleInitialize failed! Result was: `OLE_E_WRONGCOMPOBJ`");
-                    } else if ole_init_result == RPC_E_CHANGED_MODE {
+                    if let Err(error) = ole_init_result {
                         panic!(
-                            "OleInitialize failed! Result was: `RPC_E_CHANGED_MODE`. \
-                            Make sure other crates are not using multithreaded COM library \
-                            on the same thread or disable drag and drop support."
+                            "OleInitialize failed! Result was: `{:?}`{}",
+                            error.code(),
```

Comment:
If you drop `.code()` here this should show a pretty error message with `HRESULT` number and stringified error, instead of just the `HRESULT` constant.